### DISCUSSION TITLE
Amazon SES email configuration

### DISCUSSION THREAD

**Discussion by shruti3885** [pimcore/pimcore#10155] [Q&A / Support] [answered]
With Pimcore v10, i need to configure Amazon SES email settings:
Tried with below options:
main: smtp://host:port?encryption=tls&auth_mode=login&username=uname&password=pwd
main: smtp://uname:pwd@host:port
but not working, it throws following error:
![image](https://user-images.githubusercontent.com/89530194/130805821-3b3b36f8-bf9b-4f65-b334-6999b6afaa94.png)

Any idea?

**Comment by brusch** [comment, marked as answer]
See https://symfony.com/doc/current/mailer.html#using-a-3rd-party-transport
